cli/flox/src/commands/services/status.rs

```diff
@@ -127,22 +139,37 @@ impl Display for ProcessStatesDisplay {
             name_width_min,
             self.0.iter().map(|proc| proc.name.len()).max().unwrap_or(0),
         );
-        // Max value based on the possible states in:
-        // https://github.com/F1bonacc1/process-compose/blob/v1.9.0/src/types/process.go#L125-L137
-        let status_width = 12;
+
+        let status_width = self
+            .0
+            .iter()
+            .map(|proc| {
+                if let Some(exit_code) = proc.exit_code {
+                    format!("{} ({})", proc.status, exit_code).len()
+                } else {
+                    proc.status.len()
+                }
+            })
+            .max()
+            .unwrap_or(6);
 
         writeln!(
             f,
             "{:<name_width$} {:<status_width$} {:>8}",
-            "NAME", "STATUS", "PID"
+            "NAME", "STATUS", "PID",
         )?;
+
         for proc in &self.0 {
             writeln!(
                 f,
                 "{:<name_width$} {:<status_width$} {:>8}",
                 proc.name,
-                proc.status,
-                proc.pid_display()
+                if let Some(exit_code) = proc.exit_code {
+                    format!("{} ({})", proc.status, exit_code)
+                } else {
+                    proc.status.clone()
+                },
```

Comment:
This could be refactored to `proc.status_display()` since we do the same logic for calculating `status_width`.